.github/workflows/run-kata-coco-tests.yaml

```diff
@@ -181,6 +181,8 @@ jobs:
       - name: Run tests
         timeout-minutes: 30
         run: bash tests/integration/kubernetes/gha-run.sh run-tests
+        env:
+          PULL_TYPE: ${{ matrix.pull-type }}
```

Comment:
This isn't needed because `PULL_TYPE` is already exported on all steps of the job.

Reply:
Fixed.